RoboJackets/robocup-software
# Added Black Formatter to "make pretty-lines" Command

## Description
Added the formatter Black and its associated config file (pyproject.toml)

## Steps to test
1. Make sure black is installed by either running in a terminal `pip install black` or running `./util/ubuntu-setup ` in the repo
2. cd into the repo in a terminal and run `make pretty-lines` 

Expected result: The black formatter should be stylizing all python files excluding files in the external and build folders
